src/js/actions/hasHttpError.js

```diff
@@ -1,5 +1,5 @@
-export default function hasHttpError(json) {
-  return !json || json.httpStatus > 399 ?
-    true :
-    false
+export default function checkHttpError(json) {
```

Comment:
Do you think we should update the filename here too? And maybe the imports then too, just to be clear?